### DISCUSSION TITLE
Text and color?

### DISCUSSION THREAD

**Discussion by AJ-Buswell** [phildenhoff/logseq-raindrop#26] [Q&A] [answered]
Hello all.
My question is noob designed but I can't figure it out.
What separation is necessary for {text} and {color} in the settings?
Everything I try, comma, semicolon ; colon : gives me the highlights without the color.
Sorry for being such a noob. 😂

**Comment by phildenhoff** [comment, marked as answer]
Hi @AJ-Buswell! I'm sorry about that — the color property doesn't do anything. I added it to settings thinking I would add that in but I haven't done anything for it yet. I couldn't figure out what makes sense.

Since you're trying to use it, I'll ask you! Would you prefer:

1. Colors as tags: a red highlight is tagged `#Red` in Logseq
  ```
  > your highlighted text here
  #Red
  ```
2. Colors as text: a red highlight just says 
```
> your highlighted text here
(red)
```
3. Colors are namespaced tags?? Not sure if I like this:
```
> your highlighted text here
#logseq-raindrop/red
```

Or, if you have something else you think makes more sense, let me know!

**Reply by AJ-Buswell** [reply]
Thanks for your reply and no worries, there is time.
I feel your 2. Solution would be very helpful.
Plain text (red) (blue) and so on.
I use red, blue and at times green.
Very nice work BTW.
A.J
